osu.Game/Tests/Visual/OnlinePlay/BasicTestRoomManager.cs

```diff
@@ -33,10 +33,10 @@ public void CreateRoom(Room room, Action<Room> onSuccess = null, Action<string>
             room.RoomID.Value ??= Rooms.Select(r => r.RoomID.Value).Where(id => id != null).Select(id => id.Value).DefaultIfEmpty().Max() + 1;
             onSuccess?.Invoke(room);
 
-            AddRoom(room);
+            AddOrUpdateRoom(room);
         }
 
-        public void AddRoom(Room room)
+        public void AddOrUpdateRoom(Room room)
         {
             Rooms.Add(room);
             RoomsUpdated?.Invoke();
```

Comment:
This seems like it should update the room if existing now, as per the behavioural change of this method.